project.py

```diff
@@ -114,6 +114,9 @@ def _append_item_to_project(self, item):
             self._project['Issues'][-1]['closed'] = False
 
     def _people_link(self, base_url, account_id, account_name=None):
+        if account_id is None:
+            return ""
```

Comment:
Great catch! How would you feel about returning `Unknown user` like we do on line 125 below rather than an empty string? Let me know what you saw in your testing, but I think the current return value would show "by" with no text after it, and IMO "by Unknown user" would be preferable. What do you think?

Reply:
That's a good idea, I did not know where it was used when I quickly fixed it ;)

Reply:
Great, thanks!